libs/sdk-ui-dashboard/src/model/react/useDispatchDashboardCommand.ts

```diff
@@ -1,15 +1,33 @@
-// (C) 2020-2021 GoodData Corporation
+// (C) 2020-2022 GoodData Corporation
 import { useCallback } from "react";
 import { DashboardCommands } from "../commands";
 
 import { useDashboardDispatch } from "./DashboardStoreProvider";
 
 /**
  * Hook that takes command creator and returns function that will result into dispatching this command.
-
+ *
+ * The created function takes original command creators parameters as per example below.
+ *
+ * @example
+ * ```
+ * // create a dashboard command to reset AttributeFilter selection
+ * const resetAttributeFilter = useDispatchDashboardCommand(resetAttributeFilterSelection);
+ *
+ * // somewhere else in the code call the command to reset the filters
+ * **
+ * resetAttributeFilter(<dashboard-local-id>);
+ * **
+ * ```
+ *
+ * To convert {@link @gooddata/sdk-model#IPositiveAttributeFilter} or {@link @gooddata/sdk-model#INegativeAttributeFilter} to {@link @gooddata/sdk-model#IFilter} use
```

Comment:
This line is strange, IPositiveAttributeFilter is already in instance of IFilter. Also I don't think filter-specific things should be near this general-purpose hook. Maybe they should be next to the command creators instead?